admin/class-create-block-theme-admin.php

```diff
@@ -271,6 +271,62 @@ function export_theme( $theme ) {
 		die();
 	}
 
+	function create_blank_theme( $theme ) {
```

Comment:
A lot of this code looks similar to the theme export process in core, Could we reuse some of that code?